Formula/dbmate.rb

```diff
@@ -0,0 +1,17 @@
+class Dbmate < Formula
+  desc "Lightweight, framework-agnostic database migration tool"
+  homepage "https://github.com/amacneil/dbmate"
+  url "https://github.com/amacneil/dbmate/archive/v1.5.0.tar.gz"
+  sha256 "ec9a19c74ce100094f729dec1877d34097856360c88a4371959a4a5bbb4d6429"
+  head "https://github.com/amacneil/dbmate.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-s", "-o", bin/"dbmate", "."
+  end
+
+  test do
+    system bin/"dbmate", "-v"
```

Comment:
We need a better test here, this is not sufficient to ensure proper functionality. A test should exercise some minimal functionality. You can check for the existence and contents of log files, perform some simple action testing post- (and maybe pre-) conditions, and/or parse stdout/stderr for actual content indicating it works as expected.

Reply:
Thanks for the feedback. I updated with a test of actual functionality.